apps/meteor/client/components/Omnichannel/normalizeDepartments.ts

```diff
@@ -0,0 +1,20 @@
+import type { EndpointFunction } from '@rocket.chat/ui-contexts';
+
+import type { DepartmentListItem } from './hooks/useDepartmentsList';
+
+export const normalizeDepartments = async (
+	departments: DepartmentListItem[],
+	selectedDepartment: string,
+	getDepartment: EndpointFunction<'GET', '/v1/livechat/department/:_id'>,
+) => {
+	const isSelectedDepartmentAlreadyOnList = departments.find((department) => department._id === selectedDepartment);
```

Comment:
Two things. 
1. Using `.some()` is more appropriate for this scenario
2. `isSelectedDepartmentAlreadyOnList` could be a function to prevent searching the array unnecessarily.